app/services/audit/AuditService.scala

```diff
@@ -151,5 +151,5 @@ object AuditTypes extends Enumeration {
 }
 object EventData extends Enumeration {
   type Data = Value
-  val EORI, MUCR, DUCR, UCR, UCRType, MessageCode, MovementReference, SubmissionResult, Success, Failure = Value
+  val eori, mucr, ducr, ucr, ucrType, messageCode, movementReference, submissionResult, Success, Failure = Value
```

Comment:
What about Success and Failure?

Reply:
That is a value not a key. TxM wanted the keys to be `camelCase`

Reply:
I think they are only used as values, not keys in the JSON